mkdocs.yml

```diff
@@ -151,6 +151,9 @@ nav:
           - Base Application:
                 - Preparing source folder: getting-started/prepare-source-folder.md
                 - Base application: getting-started/base.md
+          - Workflows:
+                - Creating a workflow: getting-started/workflows.md
+                - Registering a workflow: getting-started/workflows/#register-workflows
```

Comment:
I think this is meant to link to a section in `workflows.md`?

Reply:
yea, it link to the section, not sure if we a new page would be better for it and want to make easy to get to.